components/Cards/FriendCard.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core";
+import Box from "@material-ui/core/Box";
+import Typography from "@material-ui/core/Typography";
+import Avatar from "@material-ui/core/Avatar";
+import AllOutIcon from "@material-ui/icons/AllOut";
+import DoneIcon from "@material-ui/icons/Done";
+import EcoIcon from "@material-ui/icons/Eco";
+import LockIcon from "@material-ui/icons/Lock";
+
+const useStyles = makeStyles({
+    box: {
+      borderRadius: "0px 15px 15px 0px",
+      // HOW TO PUT MATERIAL UI COLORS HERE
+      background: "#977BBF",
+    },
+  }),
+  FriendCard = () => {
+    const classes = useStyles();
+    return (
+      <Box className={classes.box}>
```

Comment:
Overall. Beautiful use of MUI components. Just avoid using the table component if it doesn't contain structured data :D